webapp/src/components/expanded_view/component.tsx

```diff
@@ -813,3 +977,31 @@ const style = {
         overflow: 'auto',
     },
 };
+
+const ExpandedViewGlobalsStyle = createGlobalStyle<{callThreadSelected: boolean}>`
+    #root {
+        > #global-header,
+        > .team-sidebar,
+        > .app-bar,
+        > #channel_view .channel__wrap,
+        > #SidebarContainer {
+            display: none;
+        }
+        #sidebar-right #sbrSearchFormContainer {
+            // mobile search not supported in expanded view or expanded window
+            // TODO move to hideMobileSearchBarInRHS prop of Search component in mattermost-webapp
+            display: none;
+        }
+        .channel-view-inner {
+            padding: 0;
+        }
+        ${({callThreadSelected}) => !callThreadSelected && css`
+            .sidebar--right {
+                display: none;
+            }
+        `}
+    }
+    #sidebar-right {
+        z-index: 1001;
```

Comment:
Does it need to be so high? (I'm curious :) )

Reply:
I think I can answer that since I originally set the expanded view to use `z-index` of 1000. That was the smallest value to get it to cover everything else if I recall (including any annoying tutorial step banner or upsell).